lighthouse-core/lib/i18n/README.md

```diff
@@ -0,0 +1,283 @@
+| const UIStrings = { ... };|-+                 +---> | lighthouse-core/lib/i18n/locales/en-XL.json |
+|                           |-|                 |     +----------------------------------------------+
++-----------------------------|                 |     |                                             ||
+ +----------------------------|                 |     | lighthouse-core/lib/i18n/locales/*.json     |-<+
+  +---------------------------+                 |     |                                             || |
+                           |                    |     +----------------------------------------------| |
+                           |                    |      +---------------------------------------------+ |
+              $ yarn i18n  +--------------------+                                                      |
+                           |                                                                           |
+                           v                          ▐                       ▐    +---------------+   |
+              +------------+------+                   ▐   Google TC Pipeline  ▐ +->|  *.ctc.json   |---+
+              |  en-US.ctc.json   |  +--------------> ▐      (~2 weeks)       ▐    +---------------+
+              +-------------------+  $ g3/import….sh  ▐                       ▐ $ g3/export….sh
+```
+
+#### String Collection workflow (build time)
+
+To a typical developer, the pipeline looks like this:
+
+* LH contributor makes any changes to strings.
+
+```shell
+# collect UIStrings and bake the en-US & en-XL locales
+$ yarn i18n
+
+# Test to see that the new translations are valid and apply to all strings
+$ node lighthouse-core/scripts/build-report-for-autodeployment.js && open dist/xl-accented/index.html
+```
+
+Note: Why do `en-US` and `en-XL` get baked early?  We write all our strings in `en-US` by default, so they do not need to be translated, so it can be immediately baked without going to the translators.  Similarly, `en-XL` is a debugging language, it is an automated version of `en-US` that simply adds markers to `en` strings in order to make it obvious that something has or hasn't been translated.  So neither of these files need to go to translators to be used, and both can be used at develop-time to help developer i18n workflow.
+
+#### String Translation in Google Translation Console
+
+* Googler is ready to kick off the TC pipeline again.
+
+```shell
+# collect UIStrings (to make sure everything is up to date)
+$ yarn i18n
+
+# Extract the CTC format files to translation console
+$ sh import-source-from-github.sh
+
+# Submit CL. Wait ~2 weeks for translations
+
+# Import the translated CTC format files to locales/ and bake them
+$ sh export-tc-dump-to-github.sh
+```
+
+#### String Replacement (runtime)
+
+See [Appendix A: How runtime string replacement works](#appendix)
+
+
+
+# Writing UIStrings with LHL
+
+❗TODO(exterkamp): explain all the comments and where they go/what they become.
```

Comment:
should we leave this out/file issues for them?

Reply:
imo I was going to handle that in #9114 once it was reimported :man_shrugging:

Reply:
should we just block this PR on #9114? The implementation may still change until that lands...

Reply:
let's land the doc. 9114 can always adjust anything as need be.

Reply:
I'd like to do a review then, please, so I'm not just doing a review of it in #9114 after all

Reply:
Ooops. Sorry just saw this right after i merged. Happy to take your review and i'll do whatever followups you're thinking